mne/channels/tests/test_channels.py

```diff
@@ -348,64 +349,44 @@ def test_read_ch_adjacency(tmp_path):
     pytest.raises(ValueError, read_ch_adjacency, mat_fname)
 
 
-def _download_ft_neighbors(target_dir):
-    """Download the known neighbors from FieldTrip."""
-
-    # The entire FT repository is larger than a GB, so we'll just download
-    # the few files we need.
-    def _download_one_ft_neighbor(neighbor: _BuiltinChannelAdjacency):
-        # Log level setting must happen inside the job to work properly
-        import pooch
-
-        pooch.get_logger().setLevel("ERROR")  # reduce verbosity
-        fname = neighbor.fname
-        url = neighbor.source_url
-
-        pooch.retrieve(
-            url=url,
-            known_hash=None,
-            fname=fname,
-            path=target_dir,
-        )
-
-    parallel, p_fun, _ = parallel_func(func=_download_one_ft_neighbor, n_jobs=-1)
-    parallel(
-        p_fun(neighbor)
-        for neighbor in _BUILTIN_CHANNEL_ADJACENCIES
-        if neighbor.source_url is not None
+_CHECK_ADJ = [adj for adj in _BUILTIN_CHANNEL_ADJACENCIES if adj.source_url is not None]
+# Only test a subset of the built-in adjacency matrices to speed up tests
+# (we can trust CIs to hit these very frequently so no need to test every time!)
+_CHECK_ADJ = [
+    pytest.param(_CHECK_ADJ[ii], id=_CHECK_ADJ[ii].fname)
+    for ii in range(
+        np.random.default_rng(int(time.time())).integers(0, 10),
+        len(_CHECK_ADJ),
+        10,
     )
+]
```

Comment:
@drammock unmarking for merge-when-green because I want to make sure you're on board with this... we currently test all of the files (and there are 10's of them!) and it's slow (> 10s) and flaky, e.g.:

https://github.com/mne-tools/mne-python/actions/runs/15168379382/job/42652076408?pr=13255

So to avoid that, let's just randomly test a subset. Our CIs run frequently enough that I think we can trust the random check here to find if FT updates something and we don't know about it. And anyone working on the code can easily check all of them by commenting out this subselection line. Okay with you?

Reply:
For now I think this is OK, but I have a slight preference for something deterministic rather than random, just so we know for sure they're all getting tested regularly. So, two questions:

- would marking as `ultraslow` be enough of an improvement?
- would a new test decorator that marks tests that we only do on a weekly cron job (kinda like we do for linkcheck monthly) be better?  To me this is attractive because we really don't expect these to change ever, so no point in running even a subset of the tests on every push.

Reply:
Yeah `ultraslow` is way simpler, that way just one CI suffers (and it has some time to spare compared to others) and devs generally don't, pushed!